snippets/csharp/programming-guide/nullable-types/NullableTypesUsage.cs

```diff
@@ -47,41 +52,57 @@ private static void AccessValueOfNullableType()
                 Console.WriteLine("y is undefined");
             }
             // </Snippet3>
-        }
 
-        private static void Conversions()
-        {
             // <Snippet4>
-            int? n = null;
-
-            //int m1 = n;      // Doesn't compile.
-            int n2 = (int)n;   // Compiles, but throws an exception if n is null.
-            int n3 = n.Value;  // Compiles, but throws an exception if n is null.
+            int? z = 42;
+            if (z is int zValue)
+            {
+                Console.WriteLine($"z is {zValue}");
+            }
+            else
+            {
+                Console.WriteLine("z is undefined");
```

Comment:
This is a great example, but I wouldn't use the word "undefined". Instead, I'd say "z does not have a value" or "z is null"

Reply:
@BillWagner I've updated phrasing, also in other examples.